src/TheThingsNetwork.cpp

```diff
@@ -273,11 +275,14 @@ int TheThingsNetwork::sendBytes(const byte* buffer, int length, int port, bool c
     debugPrint(F("Successful transmission. Received "));
     debugPrint(downlinkLength);
     debugPrintLn(F(" bytes"));
-    return downlinkLength;
+    if (confirm == true)
+      return 2;
+    return 1;
   }
 
   debugPrint(F("Unexpected response: "));
   debugPrintLn(response);
+  return -3;
```

Comment:
Let's make this `-10` so that we have space to introduce more (known) error codes.